src/material-experimental/mdc-chips/chip-set.ts

```diff
@@ -90,22 +132,7 @@ export class MatChipSet extends _MatChipSetMixinBase implements AfterContentInit
    * Implementation of the MDC chip-set adapter interface.
    * These methods are called by the chip set foundation.
    */
-  protected _chipSetAdapter: MDCChipSetAdapter = {
-    hasClass: (className) => this._hasMdcClass(className),
-    // No-op. We keep track of chips via ContentChildren, which will be updated when a chip is
-    // removed.
-    removeChipAtIndex: () => {},
-    // No-op for base chip set. MatChipListbox overrides the adapter to provide this method.
-    selectChipAtIndex: () => {},
-    getIndexOfChipById: (id: string) => this._chips.toArray().findIndex(chip => chip.id === id),
-    focusChipPrimaryActionAtIndex: () => {},
-    focusChipTrailingActionAtIndex: () => {},
-    removeFocusFromChipAtIndex: () => {},
-    isRTL: () => !!this._dir && this._dir.value === 'rtl',
-    getChipListCount: () => this._chips.length,
-    // TODO(mmalerba): Implement using LiveAnnouncer.
-    announceMessage: () => {},
-  };
+  protected _chipSetAdapter: MDCChipSetAdapter;
```

Comment:
remove if possible in favor of passing directly to foundation

Reply:
It's difficult to do that, since chip-listbox also needs access to an adapter (right now it uses this._chipSetAdapter)